docs/source/user_guide/utilities.ipynb

```diff
@@ -4,14 +4,37 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Utilities\n",
-    "\n",
+    "# Utilities"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "## Configuring Logging\n",
     "\n",
     "EvalML uses [the standard python logging package](https://docs.python.org/3/library/logging.html). By default, EvalML will log `INFO`-level logs and higher (warnings, errors and critical) to stdout, and will log everything to `evalml_debug.log` in the current working directory.\n",
     "\n",
     "If you want to change the location of the logfile, before import, set the `EVALML_LOG_FILE` environment variable to specify a filename within an existing directory in which you have write permission. If you want to disable logging to the logfile, set `EVALML_LOG_FILE` to be empty. If the environment variable is set to an invalid location, EvalML will print a warning message to stdout and will not create a log file."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## System Information\n",
+    "\n",
+    "EvalML provides a command-line interface (CLI) tool prints the version of EvalML and core dependencies installed, as well as some basic system information. To use this tool, just run `evalml info` in your shell or terminal. This could be useful for debugging purposes or tracking down any version-related issues."
+   ]
```

Comment:
@angela97lin I think there's a bug somewhere -- the output shows a stack trace in the docs

![Uploading Screen Shot 2020-10-20 at 12.20.30 PM.png…]()

Reply:
My bad, I was testing out the suggestion you made. Reverted to working state here (https://evalml.alteryx.com/en/796_cli_check/user_guide/utilities.html) until we discuss.